### DISCUSSION TITLE
Is it possible for Chapter 13 without geometry shader?

### DISCUSSION THREAD

**Discussion by rongcuid** [lwjglgamedev/vulkanbook#60] [Q&A]
On a mac MoltenVk device, geometry shader isn't available.

**Comment by lwjglgamedev** [comment]
Yes, the purpose of using geometry shaders is just for illustrating how they can be used. You can achieve the same result by using regular vertx-fragment shader pair. In this case you will need to record as many drawing commands as cascade shadow maps you want to use.

I'd prefer to mantain the code as it is, because as I've said, it's a way to illustrate geometry shaders.
